### ISSUE TITLE
ImportError: cannot import name 'Handle'

### ISSUE THREAD

**Issue by StrangeTcy** [opencog/opencog#3330]
When I try to run a [simple conceptual blending example](https://github.com/opencog/opencog/blob/master/examples/conceptual_blending/blend_simple.py), it eventually outputs an error that says:

>     from opencog.atomspace import Handle
> ImportError: cannot import name 'Handle'

As I understand it, `Handle` has been removed from python binding  some time ago. Am I right in thinking that the example and probably the whole python blending part has to be re-written (updated)?

**Comment by AmeBel** [comment]
Yes, you are correct on the Handle. I had started updating a but didn't complete it https://github.com/AmeBel/opencog/tree/update-python-codes. I have no plans on completing the update in python
